lumicks/pylake/kymotracker/detail/loc_models.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from scipy import integrate
+from dataclasses import dataclass
+from typing import ClassVar
+
+__all__ = ["LocalizationModel", "GaussianLocalizationModel"]
+
+
+@dataclass(frozen=True)
+class LocalizationModel:
+    """Helper (base) class to hold refinement optimization parameters.
+
+    Parameters
+    ----------
+    pixel_size : float
+        Pixel size in microns
+    position : np.array
+        Spatial coordinates in physical units.
+    """
+
+    pixel_size: float
+    position: np.ndarray
+
+    @property
+    def pixel_coordinate(self):
+        """Return spatial coordinates in units of pixels."""
+        return self.position / self.pixel_size
```

Comment:
I'd prefer if the responsibility for positional transforms to reside in the image to reduce the risk of things going out of sync.